apps/build-onchain-apps/contracts/test/SignatureMintERC721.t.sol

```diff
@@ -23,13 +23,13 @@ contract SignatureMintERC721Test is Test {
     }
 
     // --- Utility Methods ---
-    function signMessage(bytes32 message, uint256 key) internal returns (bytes memory) {
+    function signMessage(bytes32 message, uint256 key) internal pure returns (bytes memory) {
         (uint8 v, bytes32 r, bytes32 s) = vm.sign(key, message);
         bytes memory signature = toBytes(r, s, v);
         return signature;
     }
 
-    function genKeyPair() internal returns (address, uint256) {
+    function genKeyPair() view internal returns (address, uint256) {
```

Comment:
Nit: Solidity [style guide](https://docs.soliditylang.org/en/v0.8.23/style-guide.html) mentions visibility should go before mutability
```suggestion
    function genKeyPair() internal view returns (address, uint256) {
```